icons/vlcmediaplayer.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>VLC media player icon</title><path d="M11.891 13.463c3.757 0 4.854-1.347 5.065-1.674l-1.36-4.334c-.36.34-1.327.937-3.68.937-2.292 0-3.188-.567-3.52-.91l-1.39 4.422c.228.361 1.255 1.56 4.885 1.56zM11.917 3.807a6.143 6.143 0 0 0 2.376-.507L13.435.566a.781.781 0 0 0-.406-.437S12.914 0 12.032 0s-1.055.136-1.055.136a.733.733 0 0 0-.41.43l-.884 2.815a6.77 6.77 0 0 0 2.234.426zM22.59 23.319a.395.395 0 0 0-.016-.16l-2.334-7.632a.647.647 0 0 0-.583-.431h-1.663l.2.64c-.24.485-1.488 2.38-6.303 2.38-4.6 0-5.828-1.73-6.11-2.308l.223-.712H4.343a.647.647 0 0 0-.583.431L1.426 23.16a.398.398 0 0 0-.017.15l-.002.02a1.843 1.843 0 0 0 .024.226.528.528 0 0 0 .51.445H22.06a.528.528 0 0 0 .51-.445 1.843 1.843 0 0 0 .023-.226z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>VLC media player icon</title><path d="M11.891 13.4624c3.7568 0 4.8538-1.347 5.0648-1.6739l-1.36-4.3338c-.36.34-1.3269.937-3.6798.937-2.2919 0-3.1879-.567-3.5199-.91l-1.3899 4.4218c.228.361 1.255 1.56 4.8848 1.56zm.026-9.6556a6.1427 6.1427 0 002.3759-.507L13.4349.566a.781.781 0 00-.406-.437S12.914 0 12.032 0c-.882 0-1.055.136-1.055.136a.733.733 0 00-.41.43l-.8839 2.8149a6.7697 6.7697 0 002.2339.426zM22.5896 23.318a.395.395 0 00-.016-.16l-2.334-7.6316a.647.647 0 00-.583-.431h-1.6629l.2.64c-.24.485-1.4879 2.3798-6.3027 2.3798-4.5998 0-5.8278-1.7299-6.1097-2.3079l.223-.712h-1.661a.647.647 0 00-.583.431L1.4264 23.159a.398.398 0 00-.017.15l-.002.02a1.843 1.843 0 00.024.226.528.528 0 00.51.445h20.1182a.528.528 0 00.51-.445 1.843 1.843 0 00.023-.226z"/></svg>
```

Comment:
LGTM!
Could you get it down to 3 decimals though please? 😂

Reply:
@adamrusted Fixed

Reply:
I couldn't make it work with a precision of 3, even after rebuilding from source. I had to use 4 instead.